### ISSUE TITLE
StreamReader missing argument in aiohttp > 3.6.2

### ISSUE THREAD

**Issue by pbelskiy** [pnuckowski/aioresponses#175]
Hello, thanks for awesome library, but there is bug when aiohttp > 3.6.2

```python
    def stream_reader_factory(  # noqa
        loop: 'Optional[asyncio.AbstractEventLoop]' = None
    ):
        protocol = ResponseHandler(loop=loop)
>       return StreamReader(protocol, loop=loop)
E       TypeError: __init__() missing 1 required positional argument: 'limit'

.tox/pytest/lib/python3.9/site-packages/aioresponses/compat.py:48: TypeError
```

**Comment by vchetyrkine** [comment]
Duplicate of #173 ?

**Comment by pnuckowski** [comment]
Fixed in 0.7.1
